codecov.yaml

```diff
@@ -0,0 +1,35 @@
+codecov:
+  branch: main
+  require_ci_to_pass: no
+
+comment:
+  layout: "reach,diff,flags,tree"
+  behavior: default
+  require_changes: no
+  require_base: no
+  require_head: no
+  after_n_builds: 1
+  show_carryforward_flags: true
+
+github_checks:
+  annotations: true
+
+coverage:
+  status:
+    patch:
+      default:
+        target: 70%
```

Comment:
Is 70% coverage required in libOpenflow?

Reply:
For each Patch(newly added code lines), 70% coverage is required.